uber/templates/registration/new.html

```diff
@@ -54,7 +54,7 @@
                 $("#message").html(json.message);
                 if (json.success) {
                     $("#badge_num").val("");
-                    $("#message").append(' &nbsp; <a href="#" onClick="undoSale(' + json.id + ') ; return false">Undo</a>');
+                    $("#message").append(' &nbsp; <a href="#" onClick="undoSale(\'' + json.id + '\') ; return false">Undo</a>');
```

Comment:
Last year we didn't need to quote ids since they were integers, but this year they're UUIDs so we need to surround them in quotes.